website/docs/essentials/similar.md

```diff
@@ -24,6 +24,7 @@ Run the following command:
 - `-r, --require <s>` - require a module
 - `-t, --token <s>` - an access token
 - `-h, --header <s>` - set http header (`--header 'Auth: Basic 123')
+- `--apollo-federation` - Support Apollo Federation directives (default: _false_)
```

Comment:
```suggestion
- `--federation` - Support Apollo Federation directives (default: _false_)
```

Maybe `--federation`?

Reply:
@kamilkisiela i defer to writing the tests to you for this. I am not very sure what the testing conventions are and there were no existing tests inside @graphql-inspector/loaders for me to extend